week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -1,64 +1,122 @@
+ * The input of this function is a simple Array of Strings, look for the concrete example in the code.
  *
- * All of your HTML should go inside the Div tag with the id "content".
+ * All of your HTML elements should go inside the Div tag with the id "content".
  *
+ * Hint for type of lists in HTML: https://www.w3schools.com/html/html_lists.asp
  */
-function exerciseTwo(shopping) {
+function insertShoppingList(shoppingList) {
   //Write your code in here
+  let content = document.querySelector("#content");
+  let uList = document.createElement("ul");
+  for (let i = 0; i < shoppingList.length; i++) {
+    let list = document.createElement("li");
+    list.textContent = shoppingList[i];
+    uList.appendChild(list);
+    content.appendChild(uList);
+  }
 }
 
 /**
-    I'd like to display my three favorite books inside a nice webpage!
-
-    const books = [
-        {
-            title: "The Design of Everyday Things",
-            author: "Don Norman",
-            alreadyRead: false
-        },
-        {
-            title: "The Most Human Human",
-            author: "Brian Christian",
-            alreadyRead: true
-        },
-        {
-            title: "The Pragmatic Programmer",
-            author: "Andrew Hunt",
-            alreadyRead: true
-        }
-    ];
-
-    Iterate through the array of books.
-    - For each book, create a <p> element with the book title and author and append it to the page.
-    - Use a <ul> and <li> to display the books.
-    - Add an <img> to each book that links to a URL of the book cover.
-    - Change the style of the book depending on whether you have read it (green) or not (red).
-
-    The end result should look something like this: https://hyf-js2-week1-makeme-ex1-demo.herokuapp.com/
-**/
-function exerciseThree(books) {
+ * I'd like to display my three favorite books inside a nice webpage!
+ *
+ * Iterate through the array of books passed into this function and insert each book to page as follows:
+ *   - Create a <ul> list and display each book details in <li> element.
+ *   - For each book, create a <p> element with the book title and author and append it to the page.
+ *   - Add an <img> after <p> element to each book that links to a URL of the book cover.
+ *     You should find an appropriate image to each book.
+ *   - Change the style of the book depending on whether you have read it (green) or not (red).*
+ *
+ * Find in the code what properties a book object has in the array.
+ *
+ * All of your HTML elements should go inside the Div tag with the id "content".
+ *
+ * The end result should look something like this: https://hyf-js2-week1-makeme-ex1-demo.herokuapp.com/*
+ */
+function insertBooks(books) {
   //Write your code in here
+  let content = document.querySelector("#content");
+  let uList = document.createElement("ul");
+  content.appendChild(uList);
+  for (let i = 0; i < books.length; i++) {
+    let bookList = document.createElement("li");
```

Comment:
```suggestion
    let bookItem= document.createElement("li");
```
`bookList` is a slightly problematic variable name here as it might get confused with `ul` or `ol`.  Think of something like `bookItem`